infra/analytics/metabase/main.tf

```diff
@@ -123,5 +123,8 @@ module "service" {
       schema_name = local.database_config.schema_name
     }
   }
+
+  enable_drafts_bucket = startswith(local.service_name, "api-")
```

Comment:
Simpler: set the default value to `false` and only set it to true inside of `infra/api/service/main.tf`